src/lua/log.lua

```diff
@@ -117,6 +117,7 @@ local default_cfg = {
     log               = nil,
     nonblock          = nil,
     level             = S_INFO,
+    level_modules     = {},
```

Comment:
Let's make it `nil` on default/reset. IMHO if it is empty is does not of any interest and more convenient to not to show it in  cfg at all. In other words `nil` looks better then empty table.

Reply:
Done